### PR TITLE
add extract method #7718

### PR DESCRIPTION
**Pull request by AntoniosMalak**

### CODE REVIEW

**Comment by AntoniosMalak** [comment]
This is an issue I created extract #7716
This is an issue in ToDo list Add Sorting searching, and counting functions to Numpy Frontend #1531

**Comment by AntoniosMalak** [comment]
Hi @vaatsalya123, may you review the PR.

**Comment by vaatsalya123** [comment]
Hey @AntoniosMalak sorry for the delayed response , though there are a few issues with your PR here are some of the issues with your current implementation which I have labeled for you to fix.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py [hunk 1]
```diff
@@ -332,3 +344,32 @@ def test_numpy_nanargmin(
...
+    as_variable,
+    num_positional_args,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    a, cond = broadcastables
+    helpers.test_frontend_function(
+        input_dtypes=["float", "bool"],
```
Comment (vaatsalya123) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py:365]: You are passing float but it is not an acceptable dtype in this case.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py [hunk 2]
```diff
@@ -332,3 +344,32 @@ def test_numpy_nanargmin(
...
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=a,
+        cond=cond,
```
Comment (vaatsalya123) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py:374]: The name of the argument here should be condition instead.

**Comment by vaatsalya123** [comment]
Also it seems that you are using function in this implementation which don't exist in ivy's functional API so please fix that as well or else the tests will fail.

**Comment by AntoniosMalak** [comment]
Hi @vaatsalya123, Thanks for follow up:) 
You may review it now.

**Pull request closed without merging**